src/modules/rover_pos_control/RoverPositionControl.cpp

```diff
@@ -265,6 +265,37 @@ RoverPositionControl::control_position(const matrix::Vector2f &current_position,
 	return setpoint;
 }
 
+bool
+RoverPositionControl::control_velocity(const matrix::Vector2f &current_velocity,
+				       const matrix::Vector3f &ground_speed, const position_setpoint_triplet_s &pos_sp_triplet)
+{
+	bool setpoint = true;
```

Comment:
This variable should have a name what it means. E.g. `is_setpoint_valid`.

Also, it seems to never actually be changed, so you might as well just do `return true` at the end.

Reply:
Addressed by removing the return